DSharpPlus.Interactivity/Enums/PaginationBehavior.cs

```diff
@@ -26,7 +26,7 @@ namespace DSharpPlus.Interactivity.Enums
     /// <summary>
     /// Specifies how pagination will handle advancing past the first and last pages.
     /// </summary>
-    public enum PaginationBehaviour
+    public enum PaginationBehavior
```

Comment:
You should keep the old name for PaginationBehaviour, because you don't want to introduce unnecessary breaking changes in 4.x. InteractionResponseBehavior should still have the correct name.